README.md

```diff
@@ -177,44 +177,39 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
+Implementations of this specification **MUST** mount the binding data into the consuming application container in the following location:
+
+```
+<mountPathPrefix>/<service-id>/secret/<persisted_secret>
+```
+
+Where:
+* `<mountPathPrefix>` defaults to `/platform/bindings` if not specified in the `ServiceBinding` CR via the `mountPathPrefix` element.  The environment variable `SERVICE_BINDINGS_ROOT` **MUST** be set to the value of `<mountPathPrefix>`, so that applications can always find this directory.
```

Comment:
Should `SERVICE_BINDINGS_ROOT` be available as a key-value pair in the binding secret?

Reply:
that's a good point. I think generally there's an implementation issue to be solved here:  if one entity (such as the Service Binding Operator) is processing the `ServiceBinding` CR and creating the target Secret, how does another entity (such as the Runtime Component Operator) know things such as `SERVICE_BINDING_ROOT` as well as each `<service-id>` to use.  

I think that means this second entity also needs to look into the `ServiceBinding` CR.  But another general issue is:  what about `dataMappings`, which `<service-id>` do we associate it with? CC  @sbose78